solvcon/__init__.py

```diff
@@ -89,7 +89,7 @@
 from .cmdutil import Command, go
 from .conf import env
 from .block import Block
-import_name('Table', '.mesh', may_fail=True)
+import_name('Table', '.march', may_fail=True)
```

Comment:
Yes, I plan to put all SOLVCON C++ code in "libmarch", which uses a namespace "march".  In my current plan libmarch is a header-only library, but we may have multiple Cython modules exposing different parts of it.  Right now we only have solvcon.march.